cmd/aergocli/cmd/accounts.go

```diff
@@ -93,7 +93,7 @@ var newCmd = &cobra.Command{
 				cmd.Printf("Failed: %s\n", err.Error())
 				return
 			}
-			err = ks.SaveAddress(addr)
+			// err = ks.SaveAddress(addr)
```

Comment:
I think you're right, `CreateKey` calls `addKey` which calls `save`. Let's remove this whole line instead of commenting it.

Reply:
Yes you are right. I missed. Fixed in 1ce93068f62e294417aa5ab2e3778ae47ecc39d0